ee/fe/modules/values-default.yaml

```diff
@@ -3,3 +3,4 @@ monitoringApplicationsEnabled: true
 extendedMonitoringEnabled: true
 secretCopierEnabled: true
 flantIntegrationEnabled: true
+cephCsiEnabled: true
```

Comment:
I don't think we want to enable this by default because it's too specific, do we?

Reply:
You're right, we don't want to enable by default.